R/build-favicons.R

```diff
@@ -93,6 +93,17 @@ build_favicons <- function(pkg = ".", overwrite = FALSE) {
   invisible()
 }
 
+#' Deprecated as of pkgdown 1.4.0
+#' @rdname build_favicons
+#' @export
+build_favicon <- function(...) {
+  message(
+    "`build_favicon()` is deprecated as of pkgdown 1.4.0. ",
+    "Please use `build_favicons()` instead."
+  )
+  build_favicons(...)
+}
+
```

Comment:
@hadley is this what you had in mind? Let me know if there's a better way.

Reply:
I would've left the message until the next release, but since you've done it we might as well use it 😄